operator/internal/handlers/lokistack_create_or_update.go

```diff
@@ -33,6 +33,10 @@ import (
 	ctrlutil "sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 )
 
+const (
+	caKeyKey = "service-ca.crt"
```

Comment:
Would it be better to call this variable `defaultCAKey`?

Reply:
```suggestion
	defaultCAKey = "service-ca.crt"
```